app/models/wiki.rb

```diff
@@ -44,23 +50,53 @@ class Wiki < ActiveRecord::Base
     vec vep vi vls vo vro w wa war wikipedia wo wuu xal xh xmf yi yo yue za
     zea zh zh-cfr zh-classical zh-cn zh-min-nan zh-tw zh-yue zu
   ).freeze
-  validates_inclusion_of :language, in: LANGUAGES
+  validates_inclusion_of :language, in: LANGUAGES + [nil]
+
+  MULTILINGUAL_PROJECTS = {
+    'wikidata' => 'www.wikidata.org'
+  }.freeze
 
   def base_url
-    "https://#{language}.#{project}.org"
+    if language
+      "https://#{language}.#{project}.org"
+    else
+      "https://#{MULTILINGUAL_PROJECTS[project]}"
+    end
   end
 
   def api_url
     "#{base_url}/w/api.php"
```

Comment:
If there's a comment you would find helpful here, you could add it in a separate pull request. Since this PR doesn't change `api_url` or how it is used, that would usually be out of scope for the code review.